content/apply_org.md

```diff
@@ -0,0 +1,50 @@
++++
+title = "Join the Movement"
+heading = "Join the Organization for Ethical Source"
++++
+
+If you are interested in joining the Organization for Ethical Source as a member, please fill out the form below. Please indication the membership level that you are interested in:
```

Comment:
```suggestion
If you are interested in joining the Organization for Ethical Source as a member, please fill out the form below. Please indicate the kind of membership that you are interested in:
```

Let's not talk about levels, here. These are different ways of helping. All are valid.

Reply:
Not in the form?

Reply:
That was just additional context.